.prettierrc.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Why is this file being moved from the `sdk` folder to the repo root?

Reply:
@mikeharder so the settings will apply to projects outside `sdk` as well, such as ones in `common/tools` and files such as `../../../common/config/rush/pnpm-lock.yaml` are ignored.